src/lib/strings.h

```diff
@@ -23,6 +23,34 @@ void free_string(char **s);
 /* Join strings from string list separated by the separator */
 char *string_join(const char *separator, struct list *string);
 
+/*
+ * strtoi_err: Safely convert and string to integer avoiding overflows
```

Comment:
Isn't this suppose to say "Safely convert a~~nd~~ string to integer avoiding overflows."